app/src/createExcelFiles/ExcelUtils.js

```diff
@@ -0,0 +1,23 @@
+import * as XLSX from "xlsx";
+
+export function mapToExcelArray(searchItems) {
```

Comment:
you dont need to export this function

input param can be searchItemList perhaps
mapToExcelArray is fine for the method name, though you can also name it `toExcelArray` or `toExcelResult` or something. the `map` part is implied in that way.